packages/components/src/Loader/Loader.scss

```diff
@@ -1,3 +1,28 @@
 .tc-loader {
 	display: flex;
 }
+
+.tc-app-loader-icon {
+	position: absolute;
+	width: 5em;
+	height: 7em;
+	margin-left: 14px;
+	margin-right: 10px;
```

Comment:
could you explain?

Reply:
the margin properties are set to fix the appLogo position to fill the spinner center